modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -113,6 +113,12 @@ cat "$KUBECONFIG"
 
 .. Copy the `kubeconfig` file to all master hosts and move it to `/etc/kubernetes/kubeconfig`.
 
+.. Get the CA certificate used to validate connections from the API server.
++
+----
+# oc get configmap kube-apiserver-to-kubelet-client-ca -n openshift-kube-apiserver-operator --template='{{ index .data "ca-bundle.crt" }}' > /etc/kubernetes/ca.crt
+----
+
```

Comment:
According with @tnozicka  command  , we should distribute the `ca.crt` to other masters and nodes and then restart the kubelet service.

Reply:
@sjenning Can you confirm? Do we need to add a step to copy this file to all other masters at /etc/kubernetes/ca.crt?

Reply:
@zhouying7780 I've updated the doc to add a step to copy the file to the other master hosts.

Reply:
@bergerhoffer  During test I also copy the file to other nodes hosts too.